src/main/java/org/jenkinsci/test/acceptance/controller/LocalController.java

```diff
@@ -374,10 +375,15 @@ private void diagnoseFailedLoad(Exception cause) throws IOException {
             }
 
             try {
-                Process jstack = new ProcessBuilder("jstack", String.valueOf(pid)).start();
+                File tmpFile = File.createTempFile("jstack-output", ".tmp");
+                tmpFile.deleteOnExit();
```

Comment:
Can explicitly delete it also, but I'd also leave the hook in case some error happens before deleting it.